<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DigitalEventImport_Widget</class>
 <widget class="QWidget" name="DigitalEventImport_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Digital Event Import</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="dataNameLayout">
     <item>
      <widget class="QLineEdit" name="data_name_text">
       <property name="text">
        <string>events_1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_data_name">
       <property name="text">
        <string>Event Series Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="loaderTypeLayout">
     <item>
      <widget class="QLabel" name="label_loader_type">
       <property name="text">
        <string>Format:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="loader_type_combo">
       <item>
        <property name="text">
         <string>CSV</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="loaderTypeSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stacked_loader_options">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="CSVDigitalEventImport_Widget" name="csv_event_import"/>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CSVDigitalEventImport_Widget</class>
   <extends>QWidget</extends>
   <header>DigitalTimeSeries/CSV/CSVDigitalEventImport_Widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
